tests/schema/tests.py

```diff
@@ -454,9 +454,9 @@ def test_alter_text_field_to_time_field(self):
         """
         with connection.schema_editor() as editor:
             editor.create_model(Note)
-        Note.objects.create(info='3:16')
+        Note.objects.create(info='3:16:17.4567')
         old_field = Note._meta.get_field('info')
-        new_field = TimeField(blank=True)
+        new_field = TimeField(blank=True, null=True)
```

Comment:
I wanted it because it is adding a field (as an implementation detail), and the 
error text in the first try said "you can't add non-null fields to non-empty 
tables". I want to  make sure the error message is inaccurate, and that this 
holds in this context and not just in the full `AddField` operation.